frontend/src/components/Credential.vue

```diff
@@ -57,7 +58,7 @@
     </v-row>
 
     <h3 v-if="intDoc.credentialData && !showOnlyContent" class="mb-4">
-      Credential Content
+      Credential Details
```

Comment:
there's some entries in `en.json` that were using "Credential Content" (component.issueCredential), so we should change this to use a i18n and we should make sure "Credential Content" is replaced with "Credential Details" in those other resources.